Frameworks/CoreText/CTLine.mm

```diff
@@ -229,11 +228,12 @@ void CTLineDraw(CTLineRef lineRef, CGContextRef ctx) {
 
     // Translate by the inverse of the relativeXOffset to draw at the text position
     CGPoint relativePosition = { -line->_relativeXOffset, 0 };
-    for (size_t i = 0; i < [line->_runs count]; ++i) {
-        _CTRun* curRun = [line->_runs objectAtIndex:i];
+    CFIndex count = CFArrayGetCount(line->_runs);
+    for (CFIndex i = 0; i < count; ++i) {
+        __CTRun* curRun = const_cast<__CTRun*>(static_cast<CTRunRef>(CFArrayGetValueAtIndex(line->_runs, i)));
```

Comment:
can't due to the address of usage below.
 &curRun->_dwriteGlyphRun

---
In reply to: [111454096](https://github.com/Microsoft/WinObjC/pull/2511#discussion_r111454096) [](ancestors = 111454096)